### ISSUE TITLE
Allow to specify empty properties in launch profile

### ISSUE THREAD

**Issue by laeubi** [eclipse-m2e/m2e-core#412]
currently one need to enter at least one character when defining a property
![grafik](https://user-images.githubusercontent.com/1331477/142751600-19a50d0e-428d-48e8-a4a5-57ec6dcbbfbf.png)
but there are some properties that must only exits but don't take a value (e.g. https://github.com/jcgay/maven-profiler#usage )

**Comment by HannesWell** [comment]
That's a thing I also find annoying.
I have some more ideas/plans (for the mid-distant future) to improve the Launch-Config management of Maven-builds, especially the specification of properties, that also include this point (as well as something similar that you have suggested in #368). So if you can wait some time I can take care of this.

At the moment my workaround is to append such 'flag' properties in the `Goals` section using `-D<prop-name>`

**Comment by laeubi** [comment]
Yep, this is just a reminder as I sometimes notice it, then think it should be reported but then instantly forget it :-)
A quick-fix might be to simply remove the requirement for having a non empty value
